app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/EoriLanguageController.scala

```diff
@@ -28,6 +28,8 @@ class EoriLanguageController @Inject() (config: Configuration, languageUtils: La
 
   override protected def fallbackURL: String =
     "/customs-enrolment-services/register" //This will be always register for cds we might need to add a route for fallback cannot be dynamic
+
+  // TODO this is used just in tests and lang argument is not used in this method
```

Comment:
Could just get rid of this can get tests to use EoriLanguageController.routeToSwitchLanguage directly?

Reply:
Yes, I'll do it

Reply:
Done